spec/trimmer_spec.rb

```diff
@@ -0,0 +1,19 @@
+require './decorator'
+require './trimmer_decorator'
+
+describe TrimmerDecorator do
+  let(:nameable) { double('nameable') }
+  let(:decorator) { TrimmerDecorator.new(nameable) }
+
+  describe '#correct_name' do
+    it 'returns the correct trimmed name' do
+      allow(nameable).to receive(:correct_name).and_return('maximiliano')
+      expect(decorator.correct_name).to eq('maximilian')
+    end
+
+    it 'returns the full name if it is less than or equal to 10 characters' do
+      allow(nameable).to receive(:correct_name).and_return('john')
+      expect(decorator.correct_name).to eq('john')
+    end
+  end
+end
```

Comment:
- [ ] [OPTIONAL] Great job team, there are unit tests for all the methods in all the classes and they all pass, however, there is a class `trimmer_decoractor.rb`, and its corresponding test file should be `trimmer_decorator_spec.rb`. Let's resolve this because this convention helps organize and locate the test files associated with a particular class or module.
<img width="860" alt="Screenshot 2023-06-28 at 23 44 04" src="https://github.com/smunoz1988/school-library/assets/58692590/529548f1-a026-40ba-8fb3-663a94773fd3">

Reply:
@Whoistolu  / @danielamoreno699 

Done! thanks for the feedback!